src/deck-options.md

```diff
@@ -386,193 +346,149 @@ The options are:
+before optimizing the parameters, if you'd like to change which cards
+are used for optimization.
 
-If you wish to visualize how FSRS would change your schedule without altering
-your workload, there are two ways you can do so:
+You can also optimize the parameters for all of your presets at once, by clicking on the
+down arrow in the top right, then choosing "Optimize all presets".
 
-- Enable FSRS without rescheduling, and compare the interval and stability
-  graphs. The interval graph will show the current intervals of cards; the stability
-  graph will show the intervals FSRS would give cards if the desired retention is 0.9.
-- Create a backup, enable FSRS with rescheduling, check the future due graph, and then
-  undo or restore from the backup.
+**Evaluate FSRS Parameters**
 
-**Optimize FSRS parameters**
+You can use the "Evaluate" button to see metrics that show how well the parameters fit your review history. Smaller numbers
+indicate a better fit to your review history.
 
-The FSRS optimizer uses machine learning to learn your memory patterns
-and find parameters that best fit your review history. To do this, the optimizer
-requires several reviews to fine-tune the parameters.
+Log loss doesn't have an intuitive interpretation. RMSE (bins) can be
+interpreted as the average difference between the predicted probability
+of recalling a card (R) and the actual
+probability measured from your review history. For example, RMSE=5% means that, on average, FSRS
+is off by 5% when predicting R.
 
-If you have less than 1,000 reviews, you can use the default parameters that
-are already entered into the "FSRS parameters" field. Even with the default
-parameters, FSRS should work well for most users.
+Note that log loss and RMSE (bins) are not perfectly correlated,
+so two decks may have similar RMSE values but very different log-loss values, or the other way around.
 
-Once you've done 1000+ reviews in Anki, you can use the `Optimize` button to
-analyze your review history, and automatically generate parameters that are
-optimal for your memory and the content you're studying. Parameters are
-preset-specific, so if you have decks that vary wildly in difficulty, it
-is recommended to assign them separate presets, as the parameters for easy
-decks and hard decks will be different. There is no need to optimize your
-parameters frequently - once every few months is sufficient.
+By default, log loss and RMSE (bins) are calculated from all decks using the current preset. You can optionally [adjust the search](/searching.md) before evaluating the parameters, if you'd like to change which cards are used for evaluation.
 
-By default, parameters will be calculated from the review history of all
-decks using the current preset. You can optionally adjust the search
-before calculating the parameters, if you'd like to alter which cards
-are used for optimizing the parameters.
+### Ignore Reviews Before
```

Comment:
Depending on when this will be merged, it may be better to rename it to "Ignore cards reviewed before", since it will be renamed in the next release

Reply:
I'll keep these in mind.

Reply:
Suggest we just change it before we forget; a temporary mismatch shouldn't be that confusing.